tacs/pymeshloader.py

```diff
@@ -894,3 +894,59 @@ def idMap(self, fromIDs, tacsIDDict):
                 return tacsIDDict[fromIDs]
             else:
                 return -1
+
+    @property
+    def struct_ids(self):
+        """
+        get list of tacs_ids for each nastran node owned by this processor
+        nastran_node = array_idx + 1
+        tacs_idx = output id
+        nastran_node - 1 => tacs_idx owned by this proc
+        """
+
+        if self.comm is None:
+            struct_ids = self._get_local_struct_ids()
+        else:
+            struct_ids = self._get_all_struct_ids()
+        return struct_ids
+
+    def _get_local_struct_ids(self):
```

Comment:
Use camel case for method names to stay consistent with the rest of code here (i.e. `_getLocalNodeIDs`)